hoomd/md/methods/methods.py

```diff
@@ -708,6 +708,48 @@ def _attach(self):
         super()._attach()
 
 
+class DisplacementCapped(NVE):
+    r"""Modified constant volume, constant energy dynamics with a cap on motion.
+
+    The method employs a maximum displacement allowed each time step. This
+    method can be helpful to relax a system with too much overlaps without
+    "blowing up" the system.
+
+    Args:
+        filter (hoomd.filter.filter_like): Subset of particles on which to
+            apply this method.
+        maximum_displacement (hoomd.variant.variant_like): The maximum
+            displacement allowed for a particular timestep.
+
+    `MotionCapped` integrates integrates translational and rotational degrees of
+    freedom using modified microcanoncial dynamics. See `NVE` for the basis of
+    the algorithm.
+
```

Comment:
Add a warning block stating that displacement capped dynamics are not energy or momentum conserving.